build.gradle.kts

```diff
@@ -12,27 +12,47 @@ plugins {
 
 group = "cn.nukkit"
 version = "1.0-SNAPSHOT"
-description = "Nuclear powered server software for Minecraft: Bedrock Edition"
+description = "Nuclear powered server software for Minecraft Bedrock Edition"
 
 repositories {
     mavenLocal()
     mavenCentral()
     maven("https://repo.opencollab.dev/maven-releases")
     maven("https://repo.opencollab.dev/maven-snapshots")
+    maven("https://repo.mznt.eu/snapshots") //TODO
```

Comment:
What is the repo? We shouldn't be depending on repos outside Maven, Gradle and our own.

Reply:
This is used for a specific block state updater build. I plan to get rid of this before merged as it is not needed anymore.